backend/app/usecase/repository/usershortlink.go

```diff
@@ -6,4 +6,5 @@ import "github.com/short-d/short/backend/app/entity"
 type UserShortLink interface {
 	CreateRelation(user entity.User, shortLink entity.ShortLink) error
 	FindAliasesByUser(user entity.User) ([]string, error)
+	FindAliasByUser(user entity.User, alias string) (bool, error)
```

Comment:
Not sure about this one, but FindAliasByUser sounds confusing as I would expect an alias of type string to be returned. What do you think?

Reply:
I think that this is a bit confusing as well. What about `IsAliasOwnedByUser`?